src/main/java/org/opencds/cqf/tooling/terminology/CodeSystemLookupDictionary.java

```diff
@@ -188,7 +191,7 @@ public static  String getUrlFromName(String name) {
             case "UBTOB": return "http://terminology.hl7.org/CodeSystem/nubc-UB92";
             case "POS": return "http://terminology.hl7.org/CodeSystem/POS";
             case "PYXIS": return "http://content.alphora.com/fhir/dqm/CodeSystem/Pyxis";
-            case "CDCREC": return "http://terminology.hl7.org/CodeSystem/PHRaceAndEthnicityCDC";
+            case "CDCREC": return "https://terminology.hl7.org/3.1.0/NamingSystem-CDCREC.html";
```

Comment:
Per THO link: https://terminology.hl7.org/CodeSystem-CDCREC.html
The URL here should be: urn:oid:2.16.840.1.113883.6.238